NightscoutServiceKit/NightscoutService.swift

```diff
@@ -375,17 +368,153 @@ extension NightscoutService: RemoteDataService {
         })
     }
     
-    enum NotificationValidationError: LocalizedError {
-        case missingOTP
+    public func handleRemoteNotification(_ notification: [String: AnyObject]) async throws {
+        let commandSource = try commandSource(notification: notification)
+        await commandSource.handleRemoteNotification(notification)
+    }
+    
+    public func processPendingRemoteCommands() async throws {
+        try await commandSourceV2.processPendingRemoteCommands()
+    }
+    
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

This will be updated to return the commandSourceV1 until the remote 2 prs.